From foo@baz Tue Apr  9 12:12:43 2002
Date: Thu, 16 Mar 2006 11:23:21 -0700
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: sysfs: don't export dir symbols

These functions should only be used by the kobject core, and if any
driver tries to use them, bad things happen.  Unexport them to try to
prevent this from happening.


Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 fs/sysfs/dir.c |    4 ----
 1 file changed, 4 deletions(-)

--- gregkh-2.6.orig/fs/sysfs/dir.c
+++ gregkh-2.6/fs/sysfs/dir.c
@@ -511,7 +511,3 @@ struct file_operations sysfs_dir_operati
 	.read		= generic_read_dir,
 	.readdir	= sysfs_readdir,
 };
-
-EXPORT_SYMBOL_GPL(sysfs_create_dir);
-EXPORT_SYMBOL_GPL(sysfs_remove_dir);
-EXPORT_SYMBOL_GPL(sysfs_rename_dir);
